### PR TITLE
First style QA

### PR DESCRIPTION
**Pull request by rendude**
Reviewed by Becki. Significant style changes to the index page and symptom pages. All style changes and no logic change.

### CODE REVIEW

File: src/_borders.css [hunk 1]
```diff
@@ -28,6 +28,12 @@
   .bl { border-left-style: solid; border-left-width: 1px; }
   .bn { border-style: none; border-width: 0; }
 
+  @media (min-width: 48rem) {
+    .bt-double-column {
+      border-top-style: solid; border-top-width: 1px; 
+    }
+  }
+
```
Comment (misscs) [inline src/_borders.css]: What's the use case for this?

File: src/_columns.css [hunk 2]
```diff
@@ -11,6 +11,13 @@
     column-gap: var(--spacing-large);
     column-width: 50%;
   }
+
+  .col5-m {
+    column-count: 2;
+    column-gap: var(--spacing-extra-extra-large);
+    column-width: 50%;
+  }
+
```
Comment (misscs) [inline src/_columns.css]: This class name is a bit confusing. The pattern here is that the number refers to the number of columns. It seems that what we need/want to do is just change the column gap size, right? Can we make that change in `col2-m` or is that class also used?

File: src/_columns.css [hunk 3]
```diff
@@ -19,5 +26,11 @@
     column-gap: var(--spacing-large);
     column-width: calc(100% / 3);
   }
+
+  .col5-l {
+    column-count: 3;
+    column-gap: var(--spacing-extra-extra-large);
+    column-width: calc(100% / 3);
+  }
```
Comment (misscs) [inline src/_columns.css]: Same comment here as for `col5-m`. If we need a second class to reflect a wider column gap, then that's what the class name needs to indicate.

File: src/_typography.css [hunk 4]
```diff
@@ -14,22 +14,22 @@
 
 /* Measure is limited to ~66 characters */
 .measure {
-  max-width: 30em;
+  max-width: 30rem;
```
Comment (misscs) [inline src/_typography.css]: Why did these change from em to rem?

File: src/_buttons.css [hunk 5]
```diff
@@ -6,6 +6,7 @@
   Use .btn class on <a> or <button>.
 
 */
+@import "_type-scale.css"
```
Comment (misscs) [inline src/_buttons.css]: Not sure we need this, but I'll let you explain why you added it :)

**Comment by misscs** [comment]
@rendude Nice work. It's great to see the patterns emerging after the design edits. I am seeing more ways that we can develop a much tighter css library, too. Let me know if you have questions about any of my comments :)

File: src/_buttons.css (hunk 5 above)
Comment (rendude) [reply src/_buttons.css]: Removed, at first I thought we had to import it to use var(--sizes). Forgot to remove it.

File: src/_typography.css (hunk 4 above)
Comment (rendude) [reply src/_typography.css]: Thought it was confusing to use em and having to know the parent's width to then recalculate. If we are using measure, thought it was less confusing to use rem.

File: src/_borders.css (hunk 1 above)
Comment (rendude) [reply src/_borders.css]: The first h2 border for the welcome section needed the border to be visible on desktop but not on mobile.

File: src/_columns.css (hunk 2 above)
Comment (rendude) [reply src/_columns.css]: Oh right.. looks like col2-m and col3-l are both not used anywhere else. Adjusted the gap on those two classes instead.

**Comment by misscs** [comment]
Can we just remove the border using the standard border classes then?

**Comment by misscs** [comment]
ems are traditionally used in this case over rems. i’m not sure i follow what you mean about recalculating. would be great to stick to conventions and keep these ems.

File: src/_columns.css [hunk 6]
```diff
@@ -7,27 +7,14 @@
 
 @media (--breakpoint-medium) {
   .col2-m {
-    column-count: 2;
-    column-gap: var(--spacing-large);
-    column-width: 50%;
-  }
-
-  .col5-m {
```
Comment (misscs) [inline src/_columns.css]: Make sure this class gets removed in the other repo

Comment (rendude) [reply src/_columns.css]: Sure did :)

**Pull request merged**